realm/realm-library/src/main/java/io/realm/RealmObject.java

```diff
@@ -354,6 +365,30 @@ public final boolean load() {
     }
 
     /**
+     * Adds a change listener to a RealmObject.
```

Comment:
"Adds a change listener to a RealmObject that will be triggered if any field or referenced object is changed" ?